src/frontend/cypress.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'cypress';
+import dotEnv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
+import { resolve } from 'path';
+
+const myEnv = dotEnv.config({
+  path: resolve(__dirname, '../../.env'),
+});
+dotenvExpand.expand(myEnv);
```

Comment:
Oh, without this a value like `FRONTEND_ADDR=0.0.0.0:${FRONTEND_PORT}` in `.env` will remain as the literal string and not become `0.0.0.0:8080`? I didn't know this